.github/workflows/taiko-client--docker.yml

```diff
@@ -51,3 +51,7 @@ jobs:
           push: true
           tags: ${{ steps.meta.outputs.tags }}
           labels: ${{ steps.meta.outputs.labels }}
+
+      - name: hive test
```

Comment:
could we make this change in a separate PR and mark it with feat(repo): ?

Reply:
When a new Docker image is generated, the Hive test needs to be triggered. I haven't found an effective way to achieve this yet. Do you have any better suggestions?

Reply:
We can keep this solution, I just want to migrate the change to a separate PR for documentation clarity!

Reply:
`taiko-client--hive_test.yml` is added.